recipe/meta.yaml

```diff
@@ -1,12 +1,12 @@
-{% set version = "1.11.4" %}
+{% set version = "1.10.7" %}
```

Comment:
Same comment as #32 
Please explain the reasoning to why we should maintain an old version.

Reply:
@ocefpaf : I don't want you to maintain the older version.    Django-1.10.7 was not on conda-forge, and I was trying to build that. Could you please tell me the appropriate way to build an older version that is not available on conda-forge.